server_development/topics/auth-spi.adoc

```diff
@@ -1199,7 +1199,7 @@ Actually {project_name} has 2 default implementations of client authentication:
 
 Traditional authentication with client_id and client_secret::
   This is default mechanism mentioned in the https://openid.net/specs/openid-connect-core-1_0.html[OpenID Connect]                                or https://datatracker.ietf.org/doc/html/rfc6749[OAuth2] specification and {project_name} supports it since it's early days.
-  The public client needs to include `client_id` parameter with its ID in the POST request (so it's defacto not authenticated) and the confidential client needs to include `Authorization: Basic` header with the clientId and clientSecret used as username and password.
+  The public client needs to include `client_id` parameter with its ID in the POST request (so it's de facto not authenticated) and the confidential client needs to include `Authorization: Basic` header with the clientId and clientSecret used as username and password.
```

Comment:
Should we  "italicized" ?

Reply:
@dreamcrash, what do you mean by "italicized" ?

Reply:
Should we make the word 'de fact' italic _de fact_. Normally, that it is italicized since it comes from latim.

Reply:
I do not know, but in https://www.washingtonpost.com/world/2022/06/24/afghanistan-endures-yet-another-deadly-shock/ the term is not in italic.

Reply:
Yep, you can have links both ways.

"If you’re following a style guide, you should check its advice on italicizing non-English words. For writers without a style guide, the question of when to use italics for foreign words and phrases is largely a matter of choice. As a rule of thumb, if the word isn’t in the dictionary and you’re not sure your readers will know what it means, then you should italicize it."

I would say let us not be too pedantic :D and keep it like it is (_i.e.,_ not italicized)